packages/manager/src/features/CloudPulse/shared/CloudPulseResourcesSelect.tsx

```diff
@@ -100,6 +119,26 @@ export const CloudPulseResourcesSelect = React.memo(
       // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [resources, region, xFilter, resourceType]);
 
+    // selected resources will appear at the top in the autcomplete popper
```

Comment:
@kmuddapo , here we are achieving to show selected options on top of the dropdown, we can have it if you like it based on demo